twitter/views.py

```diff
@@ -9,18 +9,41 @@
 from django.utils import timezone
 from django.views import generic
 
-from .models import Tweet
+from .models import Tweet,Follow
 from .forms import TweetForm
 
 # Create your views here.
 #ツイートの一覧表示
 @login_required
 def top(request):
-#    follow_list = Follow.objects.filter(user=request.user).values_list('followed_user', flat=True)
-#    tweet_list = Tweet.objects.filter(Q(user=request.user) | Q(user__in=follow_list))
-    tweet_list = Tweet.objects.filter(user=request.user).order_by('created_time').reverse()[:20]
+    follow_list = Follow.objects.filter(user=request.user).values_list('followed_user', flat=True)
+    tweet_list = Tweet.objects.filter(Q(user=request.user) | Q(user__in=follow_list)).order_by('created_time').reverse()[:20]
     return render(request, 'twitter/top.html', {'tweet_list': tweet_list})
 
+#他の人のツイート（自分含む）
+def other_user(request, other_user_id):
+    other_user = User.objects.get(id=other_user_id)
+    tweet_list = Tweet.objects.filter(user=other_user).order_by('created_time').reverse()[:20]
+    follow = Follow.objects.filter(user=request.user, followed_user=other_user_id).exists()
+    return render(request, 'twitter/other_tweet.html', {'other_user': other_user, 'tweet_list': tweet_list, 'follow': follow})
+
+#フォロー機能
+def user_followed(request, other_user_id):
+    other_user = User.objects.get(id=other_user_id)
+    follow, created = Follow.objects.get_or_create(
+        user=request.user,
+        followed_user=other_user_id
+        )
+    if created is True:
```

Comment:
以下のように記述しても良いと思いますが如何ですか？
```suggestion
    if created:
```

Reply:
修正しました